libs/sdk-backend-tiger/src/convertors/fromBackend/dateFormatting/defaultDateFormatter.ts

```diff
@@ -1,32 +1,40 @@
 // (C) 2020-2022 GoodData Corporation
 import format from "date-fns/format";
-import { enUS, de, es, fr, ja, nl, pt, ptBR, zhCN, ru } from "date-fns/locale";
+import { enUS, enGB, cs, de, es, fr, ja, nl, pt, ptBR, zhCN, ru } from "date-fns/locale";
 import { DateAttributeGranularity } from "@gooddata/sdk-model";
 import { UnexpectedError } from "@gooddata/sdk-backend-spi";
-import { DateFormatter } from "./types";
+import identity from "lodash/identity";
 
-const granularityFormatPatterns: {
+const defaultLocaleCode = "en-US";
+
+/**
+ * Default date format patterns for all granularities.
+ * They should be identical to the ones that backend sends in case of en-US locale.
```

Comment:
Again, some link to https://date-fns.org/docs/format or https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table would be useful here (so it's clear, which standard it uses)